databricks/koalas/groupby.py

```diff
@@ -2661,14 +2666,6 @@ def _apply_series_op(self, op, should_resolve: bool = False, numeric_only: bool
         else:
             return kser
 
-    @property
```

Comment:
Would it be fine to make "_kdf" and "_agg_columns" mutable?
I'm curious why we set them immutable initially.

Reply:
It's just providing `_kdf` and `_agg_column` instead of holding them as instance variables.